accepted/010-database.md

```diff
@@ -0,0 +1,41 @@
+# SQLite for Auraed
```

Comment:
i see no major issues upfront with this decision as long as this statement from your email remains true “  under no circumstance will either "node" in the diagram need to have awareness of each other”…..this is, to me, the anchor to this decision…if that ever changes, this ADR will need a revisit.